compass/ocean/tests/global_ocean/mesh/rrs6to18/namelist.init

```diff
@@ -4,4 +4,6 @@ config_rx1_vert_smooth_weight = 10.0
 config_rx1_slope_weight = 1e-1
 config_rx1_zstar_weight = 10.0
 config_rx1_horiz_smooth_open_ocean_cells = 40
-config_rx1_min_layer_thickness = 0.1
+config_rx1_min_levels = 8
+config_rx1_min_layer_thickness = 2.5
+config_global_ocean_minimum_depth = 20.0
```

Comment:
The combination of the first 2 should mean a minimum water-column thickness of 20 m in cavities and adjacent to them in the smoothing region.

The last one applies to the z-star region away from cavities and, in practice, means a minimum of 8 vertical layers.